docker/Makefile

```diff
@@ -44,3 +44,24 @@ calibration-down:
 		-f powergate-build-context.yaml \
 		down
 .PHONY: calibration-down
+
+
+nerpa-up: nerpa-down
+	LOTUS_IMAGE_TAG=nerpa-53c7594 \
```

Comment:
Unfortunately, there isn't a version TAG in the `ntwk-nerpa` Lotus branch (yet?).
So the used image from `textileio/lotus-build` is `nerpa-53c7594` where `53c7594` is the _sha1_ of the commit of the `lotus@ntwk-nerpa`.